src/contractor/contractor.cpp

```diff
@@ -117,6 +117,7 @@ template <typename T> inline bool is_aligned(const void *pointer)
     return reinterpret_cast<uintptr_t>(pointer) % alignof(T) == 0;
 }
 
+#if !defined(NDEBUG)
```

Comment:
I will open another PR to fix current warnings, just missed this one because it is only related to Release builds